frontend/controller/signupController.js

```diff
@@ -56,16 +68,38 @@ const BrandSignUp = async (payload, navigation,showAlert) => {
 
 
 const InfluencerSignUp = async (payload, navigation,showAlert) => {
+  const data = new FormData();
+  data.append("userName", payload.userName);
+  data.append("email", payload.email);
+  data.append("password", payload.password);
```

Comment:
same here, let's not show the password in plaintext here, either have it with eye icon or remove it